app/models/meeting_option.rb

```diff
@@ -12,4 +12,11 @@ def self.bbb_options(room_id:)
       .where(room_meeting_options: { room_id: })
       .pluck(:name, :value)
   end
+
+  def self.find_room_meeting_option_by_name(name:, room_id:)
```

Comment:
I think we might want to name this something a little simpler. Maybe `get_value` or something along those lines

Reply:
It's returning more than value, what you think about naming it `get_meeting_option`?